packages/tpetra/CMakeLists.txt

```diff
@@ -21,6 +21,38 @@ TRIBITS_ADD_OPTION_AND_DEFINE(
   ${Tpetra_ENABLE_DEPRECATED_CODE_DEFAULT}
 )
 
+# Supported Kokkos version in Trilinos
+# NOTE: When we snapshot Kokkos into Trilinos, we have to update these numbers to maintain
```

Comment:
Making a note here that we'll need to add a step during Kokkos releases to update the Kokkos version checked by Tpetra in the lines below, @crtrott @dalg24 @lucbv